python/README.md

```diff
@@ -8,6 +8,14 @@ Building Feather requires a C++11 compiler. We've simplified the PyPI packaging
 to include libfeather (the C++ core library) to be built statically as part of
 the Python extension build, but this may change in the future.
 
+To build the Python package, run:
+
+```shell
+# Symlink the C++ library for the static build
+ln -s ../cpp/src src
+python setup.py install
+```
```

Comment:
Can you put this in a new section of the README indicating it's for static build and packaging (for now)? I'd like to figure out how to move to library builds in a way that's easy for people to install, if possible. I only did this as a temporary measure until something more sustainable can be arranged